examples/todo/src/controllers/todo.controller.ts

```diff
@@ -15,11 +15,13 @@ import {
   post,
   put,
   requestBody,
+  tags,
 } from '@loopback/rest';
 import {Todo} from '../models';
 import {TodoRepository} from '../repositories';
 import {GeocoderService} from '../services';
 
+@tags(['TodoController'])
```

Comment:
@mschnee , why is the todo.controller.ts file in `examples` being changed? I think your controller classes in your mocha tests are sufficient to test the decorator.

Reply:
I was looking through the documentation and some other PRs to determing best practices for updating examples, but I agree, the documentation and tests are quite clear.  I'll clean this up.

What's the best way forward here on namespacing the decorators?  My first thought is to maintain the current exports (for maintenance), but also re-export them all under the new namespace as well, as they're related (for consistency):
```ts
export * from './api.decorator';
export * from './operation.decorator';
export * from './parameter.decorator';
export * from './request-body.decorator';
export * from './tags.decorator';

import {api} from './api.decorator';
import {del, get, operation, patch, post, put} from './operation.decorator';
import {param} from './parameter.decorator';
import {requestBody} from './request-body.decorator';
import {tags} from './tags.decorator';

export const oas = {
  api,
  operation,
  method: {
    get,
    post,
    del,
    patch,
    put,
  },
  param,
  requestBody,
  tags,
};
```
```

Reply:
+1 to keep direct exports for most of the decorators.

```ts
export const oas = {
  tags,
  // The rest names are optional
  api,
  operation,
  get,
  post,
  del,
  patch,
  put,
  param,
  requestBody,
};
```

Reply:
Awesome, I'll do that :+1: